ctl/string.h

```diff
@@ -51,7 +49,7 @@ class string
     using const_iterator = const char*;
     static constexpr size_t npos = -1;
 
-    ~string() /* noexcept */;
+    ~string() noexcept;
```

Comment:
Destructors are noexcept by default, I would rather leave this commented.

Reply:
Done.